tests/unit/s2n_self_talk_nonblocking_test.c

```diff
@@ -259,8 +263,22 @@ int test_send(int use_tls13, int use_iov, int prefer_throughput)
         /* Run the client */
         const int client_rc = !use_iov ? mock_client(&io_pair, blob.data, data_size) : mock_client_iov(&io_pair, iov, iov_size);
 
-        EXPECT_SUCCESS(s2n_io_pair_close_one_end(&io_pair, S2N_CLIENT));
-        _exit(client_rc);
+        /* Cleanup */
```

Comment:
At minimum this cleanup code should probably be in a function, since the iov cleanup logic so confusing.